src/Features/Core/Portable/Completion/CompletionTrigger.cs

```diff
@@ -28,22 +29,19 @@ internal CompletionTrigger(CompletionTriggerKind kind, char character = (char)0)
         /// <summary>
         /// The default <see cref="CompletionTrigger"/> when none is specified.
         /// </summary>
+        [Obsolete("Do not use.", error: true)]
```

Comment:
I can do that.  part o the reason i wanted to make this change was to force people to have to think about what they were doing.  But i think it's acceptably reasonable to have hte 'Default' and having it be set to 'Invoke'.